images/fcos-bastion-image/root/usr/lib/systemd/system/bind9.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Creates the bind9 name server service
-Wants=network-online.target
+Wants=network-online.target configure-podman-v6-network.service
```

Comment:
There's no need to add this here IMO. Can we rather try setting the one-shot unit as `Before=network-online.target`?

Finally, is this the only service that requires enablement for the ipv6 network?

Reply:
AFAIK, this is the only service

Reply:
I got wrong then. With this change, each containerrequiring access to ipv6 need to run with --network to be attached to podmanv6